pages/admin/lessons/[lessonSlug]/[pageName]/index.tsx

```diff
@@ -148,14 +125,14 @@ const Lessons = ({ data }: GetAppProps) => {
           />
         </header>
         <section>
-          <AdminLessonNav
+          <NavCard
             tabs={[
               {
-                tabName: 'modules',
-                urlPageName: 'modules'
+                text: 'modules',
+                url: `${MAIN_PATH}/${lesson.slug}/modules`,
+                isSelected: pageName === 'modules'
```

Comment:
There's much less boilerplate now, so users of this generic component no longer need to provide their own nav item component with their own styling.

An alternative way to do this would be to make it so we determine whether a tab is selected by looking at the current URL, but this would have the problem of not being customizable enough for some potential use cases where we might want the tab to be selected even if the URL is not exactly the same as the tab's URL.

Reply:
`tabSelected` is great. Make things way simpler.